pkg/blob/ref.go

```diff
@@ -212,6 +212,22 @@ func (r Ref) IsSupported() bool {
 	return ok
 }
 
+// FromBytes returns a ref object from a digest
+func FromBytes(digNameStr string, digest []byte) (Ref, error) {
```

Comment:
@bradfitz I added this function, if there is a better (already existing) way to do this lmk